test/test_quantized_tensor.py

```diff
@@ -193,6 +193,29 @@ def test_qtensor_copy(self):
         self.assertEqual(q.q_scale(), q2.q_scale())
         self.assertEqual(q.q_zero_point(), q2.q_zero_point())
 
+    def test_qtensor_clone(self):
+
+        val = 100
+        numel = 10
+
+        scale_1 = 3.2
+        zero_point_1 = 5
+        scale_2 = 0.5
+        zero_point_2 = 10
+
+        q = torch._empty_affine_quantized([numel], scale=scale_1, zero_point=zero_point_1, dtype=torch.quint8)
+        q2 = torch._empty_affine_quantized([numel], scale=scale_2, zero_point=zero_point_2, dtype=torch.quint8)
+
+        # Check whether original scale and zero_points are set correctly.
+        self.assertEqual(q.q_scale(), scale_1)
+        self.assertEqual(q.q_zero_point(), zero_point_1)
+
+        q = q2.clone()
```

Comment:
since q2 returns a new `Tensor` I think we just need to do the following part(no need to create a `q` and check the scale and zero_point on that.